main/RFXNames.cpp

```diff
@@ -1963,7 +1963,7 @@ void GetSelectorSwitchStatuses(const std::map<std::string, std::string> & option
 		std::vector<std::string>::iterator itt;
 		int i = 0;
 		std::stringstream ss;
-		for (itt = strarray.begin(); (itt != strarray.end()) && (i <= 100); ++itt) {
+		for (itt = strarray.begin(); (itt != strarray.end()); ++itt) {
```

Comment:
In the feature please make two separate pull requests because this one targets two different issues.
If for any reason we need to revert this pull request, this patch will also be removed, or visa versa

Reply:
I can do that right now if you prefer. The disadvantage is that then we have two separate pull requests that depend on each other and if you revert the other one, you break this one anyway.

Reply:
Plz leave it in.. i've seen the other pull request, and both look fine!